guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,837 @@
+{
```

Comment:
<p>Note: Please refer to the <a href="https://developers.arcgis.com/python/guide/features-module/" target="_blank">features module</a> guide to understand more about feature layers and feature collections. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>